src/rust/catnapw/futures/pushto.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//==============================================================================
+// Imports
+//==============================================================================
+
+use socket2::{
+    SockAddr,
+    Socket,
+};
+use windows::Win32::Networking::WinSock::WSAEWOULDBLOCK;
```

Comment:
Please, preappend `::` to crate-external imports:

```
use ::socket2::Socket;
use ::std::{..}
```

This avoids name clash.